packages/headless/src/features/generated-answer/generated-answer-actions.ts

```diff
@@ -115,8 +122,17 @@ export const streamAnswer = createAsyncThunk<
           )
         );
         break;
+      case 'genqa.endOfStreamType':
+        dispatch(setIsStreaming(false));
+        dispatch(
+          logGeneratedAnswerStreamEnd(
+            (JSON.parse(payload) as GeneratedAnswerEndOfStreamPayload)
+              .answerGenerated
+          )
+        );
+        break;
       default:
-        extra.logger.error(`Unknown payloadType: "${payloadType}"`);
+        extra.logger.info(`Unknown payloadType: "${payloadType}"`);
```

Comment:
It seems to me this message should be logged as a warning. Considering there is nothing a user can do to prevent this situation from happening, maybe the message should be logged only when the debug mode is enabled.

Reply:
Yeah why not. Good idea